src/lib.rs

```diff
@@ -164,8 +164,13 @@ pub struct Images {
 impl Stream {
     /// Returns the data for the CPU architecture matching the running process.
     pub fn this_architecture(&self) -> Option<&Arch> {
-        let un = nix::sys::utsname::uname();
-        self.architectures.get(un.machine())
+        nix::sys::utsname::uname()
+            // uname() doesn't normally fail, and our return type assumes it
+            // won't.  Just produce None in that case.
+            .ok()
```

Comment:
I don't feel really strongly but I'd say actually `unwrap()` seems more appropriate.  If somehow `uname` did fail (seccomp filter going awry?) having that manifest as "no architecture" would be harder to debug than a straight up crash.

Reply:
But, feel free to merge as is from my PoV too.

Reply:
Yeah, fair point.  Updated.